mint-auditor/migrations/2022-05-24-214504_create_initial/up.sql

```diff
@@ -0,0 +1,37 @@
+-- Audit data per block
+CREATE TABLE block_audit_data (
+    -- Diesel requires having a primary key and sqlite doesn't allow 64 bit primay keys, so even though
+    -- we would've wanted to use the block_index for that we can't.
+    id INT PRIMARY KEY,
+    block_index UNSIGNED BIGINT NOT NULL
+    -- Future revision would add gnosis safe data here
+);
+CREATE UNIQUE INDEX idx__block_audit_data__block_index ON block_audit_data(block_index);
```

Comment:
This doesn't work since Diesel requires the primary key to be `i32`, I tried it. The reason I went with `INT PRIMARY KEY` is based on https://www.sqlite.org/autoinc.html: `In SQLite, a column with type INTEGER PRIMARY KEY is an alias for the [ROWID](https://www.sqlite.org/lang_createtable.html#rowid) (except in [WITHOUT ROWID](https://www.sqlite.org/withoutrowid.html) tables) which is always a 64-bit signed integer.`

Reply:
I see, that's kind of frustrating on Diesel's part.